imap_processing/idex/idex_l2a.py

```diff
@@ -182,3 +188,30 @@ def time_to_mass(
     ) ** 2
 
     return best_stretch, best_shift, mass_scale
+
+
+def calculate_kappa(mass_scales: np.ndarray, peaks_2d: list) -> NDArray:
+    """
+    Calculate the kappa value for each peak.
```

Comment:
What is the "kappa" value? Might help to reference/link to somewhere or indicate best-fit parameter or something more descriptive.